### ISSUE TITLE
Next release to Pypi

### ISSUE THREAD

**Issue by erikhuck** [scikit-multilearn/scikit-multilearn#288]
It looks like the [last version of this repository](https://pypi.org/project/scikit-multilearn/#history) released to PyPi was in 2018. Looks like we're due for another release.

**Comment by florianbaer** [comment]
I agree, what is needed for this?

**Comment by erikhuck** [comment]
@ChristianSch @northern-64bit Do either of you have access to the pypi for this repo?

**Comment by ChristianSch** [comment]
Nope, sorry. @niedakh?

**Comment by northern-64bit** [comment]
> @ChristianSch @northern-64bit Do either of you have access to the pypi for this repo?

Sadly not.

**Comment by florianbaer** [comment]
@niedakh Would it be possible to share the pipy access with one of the other collaborators? Sorry to bug you even more

**Comment by niedakh** [comment]
@florianbaer sure I can, who wants to take over?

**Comment by florianbaer** [comment]
@ChristianSch @northern-64bit ?

**Comment by ChristianSch** [comment]
Nope, not me, sorry. You guys noticed https://github.com/scikit-multilearn-ng/scikit-multilearn-ng, right?

**Comment by northern-64bit** [comment]
Sorry, just saw this. @ChristianSch already mentioned https://github.com/scikit-multilearn-ng/scikit-multilearn-ng, which is the continuation, which I already started to build on. It already has some large changes and some could be breaking. For now I'll continue to maintain and improve scikit-multilearn-ng.

So should we continue to maintain this repo or scikit-multilearn-ng in the long run? What do you think? @florianbaer @niedakh
